src/flow-worker.c

```diff
@@ -98,6 +98,9 @@ static TmEcode FlowWorkerThreadInit(ThreadVars *tv, void *initdata, void **data)
         return TM_ECODE_FAILED;
     }
 
+    AppLayerRegisterCounters(tv, IPPROTO_UDP);
```

Comment:
Not sure why we should have 2 calls here for TCP and UDP. Might just as well move the ipproto inside the Register func.